src/content/en/fundamentals/performance/http2/index.md

```diff
@@ -565,15 +565,15 @@ and simple to implement correctly, and of course, enable good compression of
 HTTP header metadata.
 
 For full details of the HPACK compression algorithm, see
-<https://tools.ietf.org/html/draft-ietf-httpbis-header-compression>.
+[IETF HPACK - Header Compression for HTTP/2](https://tools.ietf.org/html/draft-ietf-httpbis-header-compression).
 
 ## Further reading:
 
 * [“HTTP/2”](https://hpbn.co/http2/){: .external }
     – The full article by Ilya Grigorik
 * [“Setting up HTTP/2”](https://surma.link/things/h2setup/){: .external }
     – How to set up HTTP/2 in different backends by Surma
-* [“HTTP/2 is here, let’s optimize!”](https://docs.google.com/presentation/d/1r7QXGYOLCh4fcUq0jDdDwKJWNqWK1o4xMtYpKZCJYjM/edit#slide=id.p19)
+* [“HTTP/2 is here, let’s optimize!”](https://docs.google.com/presentation/d/1r7QXGYOLCh4fcUq0jDdDwKJWNqWK1o4xMtYpKZCJYjM/edit#slide=id.p19) 
```

Comment:
I know this isn't your text, but as long as we're here, 'let's optimize' should actually be the start of a new sentence. Can you please take care of that?